### DISCUSSION TITLE
Ordinal spent as fee possible ?

### DISCUSSION THREAD

**Discussion by goddesslens** [ordinals/ord#1646] [General]
Is it possible for an ordinal to be accidentally spent as fee on ord wallet ? How to make sure that, that never happens ? How to freeze them ?

**Comment by raphjaph** [comment]
If you only use `ord wallet` that shouldn't be possible. As soon as you use `bitcoin-cli` to do transactions with your wallet things can go wrong. To be extra sure you could could lock them. See #1147 and #1269.

**Reply by goddesslens** [reply]
Thank you.... I made the transaction i think it is fine. But  it was stuck with 1sat/vbyte. Any idea when RBF will be live on ord wallet ? I am helpless.. even my inscribe is stuck at 1sat/vbyye
